examples/fluids/index.md

```diff
@@ -297,8 +267,9 @@ $$
 
 where $P_0$ is the atmospheric pressure.
 For this problem, we have used no-slip and non-penetration boundary conditions for $\bm{u}$, and no-flux for mass and energy densities.
-This problem can be run with:
 
-```
-./navierstokes -problem density_current
+## Running the mini-app
```

Comment:
Can we move this to the top like we have it for the solids example? At least to me it makes more sense there.

Reply:
In my head, the 'running the mini-app' section provides a quick reference that you would frequently use and the individual problem sections are a deep dive into the details that you look at less often.

Ultimately, I have a mild preference for 'running the mini-app' before the theory, but I'm fine with either order so long as the two mini-apps match.